deployment/provision_vagrant_vm.sh

```diff
@@ -82,7 +117,13 @@ apt-get -q install -y libasound2 libgconf-2-4 libgbm1 libgtk-3-0 libnss3 libx11-
 wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh --no-verbose --output-document - | sudo -H -u $USER bash
 
 # setup evap
-cd /$USER
+if [ "$docker" = true ]
+then
+    cd /evap
+else
+    cd /$USER
+fi
```

Comment:
Isn't this `/evap` in both cases?

Reply:
it should be except if the username is changed